src/istgt_lu.c

```diff
@@ -2396,10 +2403,19 @@ istgt_lu_update_unit(ISTGT_LU_Ptr lu, CF_SECTION *sp)
 				ISTGT_ERRLOG("Unable to close Unauthorised connections for LU%d\n",lu->num);
 				goto error_out;
 			}
+
+			/*
+			 * If the client is logged in then we will skip
```

Comment:
Do we know from maxtsih or igp_old that client is logged-in ? If "yes" then please amend comment so that future reader understand it clearly.